apps/src/templates/instructions/InstructionsTab.jsx

```diff
@@ -23,12 +23,18 @@ export default class InstructionsTab extends Component {
     onClick: PropTypes.func.isRequired,
     style: PropTypes.object,
     text: PropTypes.string.isRequired,
+    isLegacyDefaultTextColor: PropTypes.bool,
     teacherOnly: PropTypes.bool,
     isMinecraft: PropTypes.bool,
     isRtl: PropTypes.bool
   };
 
   render() {
+    const {isLegacyDefaultTextColor} = this.props;
```

Comment:
should this variable be named `isLegacyTextColor` instead? It's a little confusing because later on you check `isLegacyDefaultTextColor` and if it's true, set text style to `legacyText`, otherwise `defaultText`